README.md

```diff
@@ -1,3 +1,201 @@
 # Team-258-group-a-frontend
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/40f4f71c8ee744838941cb6f1b42d978)](https://app.codacy.com/gh/BuildForSDGCohort2/Team-258-group-a-frontend?utm_source=github.com&utm_medium=referral&utm_content=BuildForSDGCohort2/Team-258-group-a-frontend&utm_campaign=Badge_Grade_Dashboard)
+
+# Fumana Work
```

Comment:
Please add a space between the title and the content

Reply:
Will do